src/Plugins/PluginDiscovery.php

```diff
@@ -46,7 +46,7 @@ public function discover()
                     try {
                         $out[] = $this->getContainer()->get(PluginInfo::class, [$dir->getPathname()]);
                     } catch (TerminusException $e) {
-                        $this->logger->debug(
+                        $this->logger->warning(
```

Comment:
This should def not be a warning. Why does the Application object need to be overridden to fix Symfony's `-vvv` option?